jobs/gtfs-schedule-validator/gtfs_schedule_validator.py

```diff
@@ -38,6 +38,7 @@
 logging.basicConfig()
 
 
+# TODO: this could be merged with the RT one
```

Comment:
should it? I thought you were regretting combining `GTFSFeedExtractInfo`

Reply:
Agreed, removing.